apps/ui/src/components/ProposalTimeline.vue

```diff
@@ -0,0 +1,130 @@
+<script setup lang="ts">
+import { _t } from '@/helpers/utils';
+import { Proposal, Space } from '@/types';
+
+type ProposalTimelineValues = {
+  created?: number;
+  start: number;
+  min_end: number;
+  max_end: number;
+};
+
+type State = {
+  id: string;
+  value: number;
+};
+
+const props = defineProps<{ data: Proposal | Space }>();
+
+const labels = {
+  created: 'Created',
+  start: 'Start',
+  end: 'End',
+  min_end: 'Min. end',
+  max_end: 'Max. end'
+};
+
+const { getTsFromCurrent, getDurationFromCurrentEVM } = useMetaStore();
+
+const now = ref(parseInt((Date.now() / 1000).toFixed()));
+
+onMounted(() => {
+  const interval = setInterval(() => {
+    now.value = parseInt((Date.now() / 1000).toFixed());
+  }, 1000);
+
+  onUnmounted(() => {
+    clearInterval(interval);
+  });
+});
+
+const formatTimelineValues = (): ProposalTimelineValues => {
```

Comment:
Aren't we preferring named function over anonymous function? (I remember we had that discussion some time ago with sam)

Reply:
This is named ofcourse 🤔 Maybe you mean arrow functions? not sure whats the discussion is, which way do we go?

Reply:
moved away from arrow function for now